src/Filters/ActivityListPagination.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Filters;
+
+class ActivityListPagination
+{
+    /**
+     * @var integer $currentPage
+     */
+    public $currentPage;
+
+    /**
+     * @var integer $pageSize
+     */
+    public $pageSize;
+
+    public function setPaginationFields(array $pagination): self
+    {
+        if (!empty($pagination['page']) && filter_var($pagination['page'], FILTER_VALIDATE_INT)) {
+            $this->currentPage = (integer)$pagination['page'];
+        } else {
+            $this->currentPage = 1;
```

Comment:
FYI, if you declare the field with a default value `public $currentPage = 1;` you won't need the `else` branch ))